src/pages/validator/amplifier/gmp.mdx

```diff
@@ -0,0 +1,95 @@
+```
+
+### Relay the execute data to the destination chain gateway
+
+The execute data can be relayed to the destination chain’s `AxelarGateway` in a number of ways. Here’s how to do it with `node.js` using `ethers`:
+
+```js
+> await wallet.sendTransaction({to: gatewayAddress, data: "0x"+executeData, gasLimit:ethers.BigNumber.from(5e+06)}).then((tx) => tx.wait())
+```
+
+### Execute the contract
+
+Fill in the appropriate values and execute the contract. The `command_id` can be found in the response to the `get_proof` query:
+
+```js
+> await contract.execute(ethers.utils.arrayify(command_id),source_chain, source_address, ethers.utils.arrayify(payload)).then((tx) => tx.wait()
+```
```

Comment:
Could also include what events / messages are emitted on success/failure on these transactions, so error handling by integrators can be done properly.